man/ensure_file_exist.Rd

```diff
@@ -0,0 +1,15 @@
+% Generated by roxygen2 (4.1.0): do not edit by hand
```

Comment:
Agreed, this file should have been tracked.  Thank you.
And thank you for rerunning `roxygenize()`.